pkg/volume/vsphere_volume/vsphere_volume.go

```diff
@@ -433,7 +433,7 @@ func (v *vsphereVolumeProvisioner) Provision(selectedNode *v1.Node, allowedTopol
 		for k, v := range labels {
 			pv.Labels[k] = v
 			var values []string
-			if k == v1.LabelFailureDomainBetaZone {
```

Comment:
we might want to keep it backwards compatible. So maybe
```
if k == v1.LabelTopologyZone{
} else if k == v1.LabelFailureDomainBetaZone {
} else
```
? What do you think

Reply:
@Jiawei0227  yes if the storage class is not updated after the upgrade we need to handle the case.

Reply:
fixed. Please see the latest commit.